app/models/event.py

```diff
@@ -260,10 +270,20 @@ def __str__(self):
 
     def __setattr__(self, name, value):
         if name == 'organizer_description' or name == 'description' or name == 'code_of_conduct':
-            super(Event, self).__setattr__(name, clean_html(clean_up_string(value)))
+            super(Event, self).__setattr__(name,
+                                           clean_html(clean_up_string(value)))
         else:
             super(Event, self).__setattr__(name, value)
 
+    @property
+    def fee(self):
+        """
+        Returns the fee as a float from 0 to 1 for this event
+
+        Is retrieved from the db using the `payment_currency`
+        """
+        return get_fee(self.payment_currency)
```

Comment:
So you're basically returning the service_fee. Why is it always between 0 to 1 ? Is it a fraction of the ticket fee that is provided to eventyay ? 
If no, then please explain. If yes, then I think the docs are extremely misleading. https://open-event-api-dev.herokuapp.com/#ticket-fees
```
"attributes": {
        "currency": "USD",
        "service-fee": 3.14,
        "maximum-fee": 5.14
      },
```

The sample response has values greater than 1 :(

Reply:
ah my mistake, just saw that. will fix it!

Reply:
fixed. can this be merged now?